src/components/job-card.jsx

```
@@ -0,0 +1,66 @@
+          )}
+        </CardTitle>
+      </CardHeader>
+      <CardContent className="flex flex-col gap-4 flex-1">
+        <div className="flex justify-between">
+          {job.company && <img src={job.company.logo_url} className="h-6" />}
+          <div className="flex gap-2 items-center">
+            <MapPinIcon size={15} /> {job.location}
+          </div>
+        </div>
+        <hr />
+        {job.description.substring(0, job.description.indexOf("."))}.
```

ESLint

[error] 'job.description' is missing in props validation

(react/prop-types)